doc/orchestrator.rst

```diff
@@ -71,100 +335,217 @@ This method is deemed ``locality based inference`` since data is local to each
 process and the ``Orchestrator`` is deployed locally on each compute host where
 the distributed application is running.
 
+-------
+Example
+-------
+This example provides a demonstration on automating the deployment of
+a colocated Orchestrator within an Experiment.
 
-To create a colocated model, first, create a ``Model`` instance and then call
-the ``Model.colocate_db_tcp`` or ``Model.colocate_db_uds`` function.
+The example is comprised of two script files:
 
-.. currentmodule:: smartsim.entity.model
+* The Application Script
+* The Experiment Driver Script
 
-.. automethod:: Model.colocate_db_tcp
-    :noindex:
+**The Application Script Overview:**
+The example application script is a Python file that contains
+instructions to create and connect a SmartRedis
+client to the colocated Orchestrator.
+Since a colocated Orchestrator is launched when the Model
```

Comment:
Is this "Since a colo...." information better shifted up into the overview describing the use of the feature instead of in the specific example?

Reply:
I am adding it to the Example overview, but if you think its beneficial, Im going to keep the summaries of the app script and driver script with modifications?

Reply:
yes it is very misplaced here for sure!